BmiCalculator/Program.cs

```diff
@@ -0,0 +1,105 @@
+﻿using System;
+
+namespace BmiCalculator
+{
+    internal static class Program
+    {
+        private static void Main(string[] args)
+        {
+            Run();
+        }
+
+        private static void Run()
+        {
+            const string welcomeMessage = "Hello World! Welcome to the Bmi Calculator";
+            Console.WriteLine(welcomeMessage);
+
+            bool isUserContinuing;
+            do
+            {
+                PrintBmi();
+                const string continueMessage = "Calculate BMI for another person? (Y/N)";
+                isUserContinuing = PromptBool(continueMessage, ConsoleKey.Y);
+                Console.WriteLine();
+            } while (isUserContinuing);
```

Comment:
A bit overkill for this task- but okay.

Reply:
Thank you for the review. Sometimes I get carried away when I get excited like I am now about programming with C#. I will try to be more on point for the next homeworks.